File: Payanak/Original/ClientApp/src/app/shared/model/sms/schedule-detail.model.ts
import {ScheduleSmsInfoModel} from './schedule-sms-info.model';
import {ContactModel} from '../contact/contact.model';

export class ScheduleDetailModel {
  id: number;
  parent: ScheduleSmsInfoModel;
  date: number;
  updatedDate: number;
  counter: number;
  user: ContactModel;
}

File: Payanak/Original/ClientModels/ScheduleDetailModel.cs
namespace Backend.ClientModels
{
    public class ScheduleDetailModel
    {
        public long Id { get; set; }
        public ScheduleSmsInfoModel Parent {get;set;}
        public long Date{get;set;}
        public long UpdatedDate{get;set;}
        public long Counter{get;set;}
        public ContactModel User{get;set;}
        
    }
}